modules/ossm-migrating-reference-unsupported-configurations.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/reference/ossm-migrating-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-migrating-reference-unsupported-configurations_{context}"]
+= Unsupported configurations in Service Mesh 3
+
+The following tables list {SMProduct} 2 `ServiceMeshControlPlane` configurations that are not supported in {SMProductName} 3. Remember that this does not necessarily mean the functionality has been removed. It can mean that the configurations can be managed separately in {SMProduct} 3, outside your `Istio` resource.
```

Comment:
```suggestion
The following tables list {SMProduct} 2 `ServiceMeshControlPlane` fields that are not supported in {SMProductName} 3. Remember that this does not necessarily mean the functionality has been removed. For example in case of addons, they are no longer managed by {SMProduct} 3 operator and must be installed as standalone applications. 
```